draft-ietf-quic-http.md

```diff
@@ -644,54 +635,29 @@ each HTTP/2 SETTINGS parameter is mapped:
-the updated parameters as soon as possible upon receipt. The values in the
-SETTINGS frame MUST be processed in the order they appear, with no other frame
-processing between values. Unsupported parameters MUST be ignored.
-
-Once all values have been processed, if the REQUEST_ACK flag was set, the
-recipient MUST emit the following frames:
-
- - On the connection control stream, a SETTINGS_ACK frame
-   ({{frame-settings-ack}}) listing the identifiers whose values were not
-   understood.
-
- - On each request control stream which is not in the "half-closed (local)" or
-   "closed" state, an empty SETTINGS_ACK frame.
-
-The SETTINGS_ACK frame on the connection control stream contains the highest
-stream number which was open at the time the SETTINGS frame was received.  All
-streams with higher numbers can safely be assumed to have the new settings in
-effect when they open.
-
-For already-open streams including the connection control stream, the
-SETTINGS_ACK frame indicates the point at which the new settings took effect, if
-they did so before the peer half-closed the stream. If the peer closed the
-stream before receiving the SETTINGS frame, the previous settings were in effect
-for the full lifetime of that stream.
+  : An integer with a maximum value of 2^32 - 1.
 
-In certain conditions, the SETTINGS_ACK frame can be the first frame on a given
-stream -- this simply indicates that the new settings apply from the beginning
-of that stream.
+#### Usage in 0-RTT
 
-If the sender of a SETTINGS frame with the REQUEST_ACK flag set does not
-receive full acknowledgement within a reasonable amount of time, it MAY issue a
-connection error ({{errors}}) of type HTTP_SETTINGS_TIMEOUT.  A full
-acknowledgement has occurred when:
+When a 0-RTT QUIC connection is being used, the client's initial requests will
+be sent before the arrival of the server's SETTINGS frame.  Clients SHOULD
+cache at least the following settings about servers:
 
- - All previous SETTINGS frames have been fully acknowledged,
+  - SETTINGS_HEADER_TABLE_SIZE
+  - SETTINGS_MAX_HEADER_LIST_SIZE
 
- - A SETTINGS_ACK frame has been received on the connection control stream,
+Clients MUST comply with cached settings until the server's current settings are
+received.  If a client does not have cached values, it MAY assume the following
+values:
 
- - All message control streams with a Stream ID through those given in the
-   SETTINGS_ACK frame have either closed or received a SETTINGS_ACK frame.
+  - SETTINGS_HEADER_TABLE_SIZE:  4,096 octets
+  - SETTINGS_MAX_HEADER_LIST_SIZE:  16,384 octets
 
+Servers MAY continue processing data from clients which conform to these
+defaults during the initial flight, even if the client behavior exceeds its
+current configuration.  If the connection is closed because these or other
+constraints were violated during the 0-RTT flight, clients MAY retry using the
+settings sent by the server on the closed connection.
```

Comment:
Silly thought:  It would be nice if we could get QUIC to guarantee delivery of a certain payload that explains the connection close.  This feels very close to the reason phrase, but isn't just human-readable text.